.github/workflows/makecommit.yml

```diff
@@ -0,0 +1,21 @@
+name: Make Generate
+on: push
+
+jobs: 
+  run:
+    name: Make Generate Template
+    runs-on: ubuntu-latest
+    steps: 
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Make
+      run: make clean && make
+
+    - name: Commit changes
+      uses: EndBug/add-and-commit@v4
+      with:
+        message: "hack: make generate"
```

Comment:
```suggestion
        message: "on push: make generate-syncset"
```

Reply:
is that our convention?
i'm always trying to structure things like "path/to/folder: what i changed"